react/modules/sample/src/configs/FormComposerConfig.js

```diff
@@ -0,0 +1,644 @@
+                    "error": "COMMON_ENTER_VALID_GEO_LOCATION",
+                    "validation": {
+                        "pattern": "^[-+]?([1-8]?\\d(\\.\\d+)?|90(\\.0+)?)\\s*,\\s*[-+]?(180(\\.0+)?|((1[0-7]\\d)|([1-9]?\\d))(\\.\\d+)?)$"
+                    }
+                },
+                "preProcess": {
+                    "convertStringToRegEx": [
+                        "populators.validation.pattern"
+                    ]
+                },
+                "isMandatory": true
+            },
+            {
+                "key": "noSubProject_ulb",
+                "type": "radioordropdown",
+                "label": "ES_COMMON_ULB",
+                "disable": true,
+                "populators": {
+                    "name": "noSubProject_ulb",
+                    "error": "WORKS_REQUIRED_ERR",
+                    "options": [],
+                    "required": true,
+                    "optionsKey": "i18nKey"
+                },
+                "preProcess": {
+                    "updateDependent": [
+                        "populators.options"
+                    ]
+                },
+                "isMandatory": false
+            },
+            {
+                "key": "noSubProject_ward",
+                "type": "radioordropdown",
+                "label": "PDF_STATIC_LABEL_ESTIMATE_WARD",
+                "disable": false,
+                "populators": {
+                    "name": "noSubProject_ward",
+                    "error": "WORKS_REQUIRED_ERR",
+                    "options": [],
+                    "required": false,
+                    "optionsKey": "i18nKey"
+                },
+                "preProcess": {
+                    "updateDependent": [
+                        "populators.options"
+                    ]
+                },
+                "isMandatory": true
+            },
+            {
+                "type": "apidropdown",
+                "label": "WORKS_LOCALITY",
+                "disable": false,
+                "populators": {
+                    "url": "/boundary-service/boundary-relationships/_search",
+                    "name": "noSubProject_locality",
+                    "error": "WORKS_REQUIRED_ERR",
+                    "params": {
+                        "boundaryType": "Locality",
+                        "hierarchyTypeCode": "ADMIN"
+                    },
+                    "required": true,
+                    "selectFun": "return data?.TenantBoundary[0]?.boundary?.sort((a, b) => a.code.localeCompare(b.code)).map(item => ({ code: item.code, name: t(`${headerLocale}_ADMIN_${item?.code}`), i18nKey: `${headerLocale}_ADMIN_${item?.code}` }))",
+                    "optionsKey": "i18nKey",
+                    "allowMultiSelect": false
+                },
+                "isMandatory": true
+            }
+        ],
+        "head": "ES_COMMON_LOCATION_DETAILS",
+        "navLink": "Project_Details"
+    },
```

Comment:
keep this config very small, add one boundary component and one other field only, remove the rest